docs/sources/setup-grafana/configure-security/planning-iam-strategy.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Planning IAM integration strategy
+menuTitle: Planning IAM integration strategy
+description: Considerations and planification before setting up Grafana for the first time.
+aliases:
```

Comment:
Because this is a new topic, you don't need an alias. You only need an alias when you want to move an existing topic to another location.

Reply:
Aliases removed!